src/components/Card.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import Proptypes from "prop-types";
+import "./card.css";
+function Card(props) {
+    const { children, className, ...rest } = props;
+
+    return (
+        <div className={` card-default ${className}`} {...rest}>
+            {children}
+        </div>
+    );
+}
+
+Card.propTypes = {
+    children: Proptypes.any,
+    className: Proptypes.string,
```

Comment:
Same suggestion as above -> Since all the Props are mandatory, you can include .isRequired too

Reply:
@smilegupta, not all are required why do you think `className` is required?

Reply:
I thought, className was a mandatory property. With className added, properties of a card might change. But i guess, it was my mis judgement.